src/test/java/org/knowm/dropwizard/sundial/SundialBundleIT.java

```diff
@@ -0,0 +1,64 @@
+package org.knowm.dropwizard.sundial;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.ws.rs.client.Client;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import io.dropwizard.client.JerseyClientBuilder;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+
+public class SundialBundleIT extends SundialBundleITBase {
+    @ClassRule
```

Comment:
This needs to be a `ClassRule`, as the Sundial scheduler is shut down when the application stops, and it cannot be restarted. This in turn means that we need one class per configuration file and run `maven-failsafe-plugin` with `reuseForks` set to `false`.